cfme/cloud/provider/__init__.py

```diff
@@ -109,6 +109,16 @@ def __init__(self, name=None, endpoints=None, zone=None, key=None, appliance=Non
         self.key = key
         self.endpoints = self._prepare_endpoints(endpoints)
 
+    def create(self, **kwargs):
+        """override BaseProvider.create() with addition of region enabling"""
+        disabled_regions = self.appliance.get_disabled_regions(self)
```

Comment:
**required** we need to sanitize the value here if we dont ensure the return value is uniform

Reply:
Moving this to fixture, but this comment holds true regardless.